nf_core/pipeline-template/{{cookiecutter.name_noslash}}/main.nf

```diff
@@ -21,27 +21,28 @@ def helpMessage() {
     nextflow run {{ cookiecutter.name }} --reads '*_R{1,2}.fastq.gz' -profile docker
 
     Mandatory arguments:
-      --reads                       Path to input data (must be surrounded with quotes)
-      -profile                      Configuration profile to use. Can use multiple (comma separated)
-                                    Available: conda, docker, singularity, awsbatch, test and more.
+      --reads [file]                Path to input data (must be surrounded with quotes)
+      -profile [str]                Configuration profile to use. Can use multiple (comma separated)
+                                    Available: conda, docker, singularity, test, awsbatch and more
```

Comment:
I would precise institute here

Reply:
I thought that would be a good idea too but how what syntax do we use to add it in? If we add `institute` then people may run it with `-profile institute`! Cant use a URL to nf-core/configs either.

`<institute>`?

Reply:
I think this will have to wait for the next release now 😅